src/java.desktop/macosx/native/libawt_lwawt/java2d/metal/MTLLayer.m

```diff
@@ -29,6 +29,7 @@
 #import "LWCToolkit.h"
 #import "MTLSurfaceData.h"
 #import "JNIUtilities.h"
+#define KEEP_ALIVE_INC 4
```

Comment:
Why only 4?
Have you experimented with lesser or greater values?

Reply:
Yes,  4 is enough to not produce extra stops for animations with relatively high frequency (20+ FPS). I've tested it on RenderPerfTest. If you have other scenarios handy I can check them too.